examples/blueprint-construct/index.ts

```diff
@@ -100,7 +100,14 @@ export default class BlueprintConstruct {
                 securityContextRunAsUser: 1001,
                 irsaRoles: ["CloudWatchFullAccess", "AmazonSQSFullAccess"]
             }),
-            new blueprints.addons.AWSPrivateCAIssuerAddon()
+            new blueprints.addons.AWSPrivateCAIssuerAddon(),
+            new blueprints.addons.JupyterHubAddOn({
+                ebsConfig: {
+                    storageClass: "gp2",
+                    capacity: "4Gi",
+                },
+                enableIngress: true,
```

Comment:
@youngjeong46 if delete fails consistently on ENI removal (or load balancer removal), I suspect it is because it happens asynchronously, so from the cluster perspective the resource is dropped, but when VPC is attempted to be deleted, it is still cleaning it up.
I suggest we set enableIngress to false here to get past the issue.